### PR TITLE
Move configuration service

### PR DESCRIPTION
**Pull request by DarkEld3r**

### CODE REVIEW

**Comment by DarkEld3r** [comment]
Finally ready for review. 😅

File: services/configuration/testnet/testnetctl.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    update) 
+        update $2
+        ;;
+    clear) 
+        clear $2
+        ;;
+    generate) 
+        generate $2 $3 $4
+        ;;
+esac
```
Comment (vitvakatu) [inline services/configuration/testnet/testnetctl.sh]: Micro-nitpick: github doesn't like no newline at the end of the file

File: services/configuration/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+serde = "1.0.10"
+serde_derive = "1.0.10"
+lazy_static = "1.0.0"
+router = "0.6.0"
+iron = "0.6.0"
+bodyparser = "0.8.0"
+params = "0.8.0"
+
+[dev-dependencies]
+exonum-testkit = {version = "0.5.0", path = "../../testkit" }
```
Comment (matklad) [inline services/configuration/Cargo.toml]: Missing space after {

File: services/configuration/testnet/testnetctl.sh (hunk 1 above)
Comment (DarkEld3r) [reply services/configuration/testnet/testnetctl.sh]: I don't like it, either. 😄

**Comment by slowli** [comment]
What do you think about the example in the `testnet` directory? IMO, it could be improved to more closely reflect best practices of Exonum deployment (e.g., using Docker containers w/ corresponding orchestration). As a possible suggestion, the example could be removed for now and then could be restored in the form of a more complete example / tutorial how to deploy Exonum in real-world environments.

**Comment by DarkEld3r** [comment]
I have created several issues for further improvements:
https://jira.bf.local/browse/ECR-696
https://jira.bf.local/browse/ECR-697

**Pull request merged**